### PR TITLE
Remove deprecated lowercase version variables

### PR DESCRIPTION
**Pull request by syedimranhassan**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

File: playbooks/legacy_certificates.yml [hunk 1]
```diff
@@ -8,7 +8,6 @@
     repo_url: "git@github.com:edx/certificates.git"
     repo_path: "/opt/wwc/certificates"
     CERTIFICATES_VERSION: "master"
-    certificates_version: "{{ CERTIFICATES_VERSION }}"
```
Comment (adzuci) [inline playbooks/legacy_certificates.yml:11]: Should we split this into two PRs so one has just the removal of the link assignments?

**Review by adzuci — COMMENTED** [review]
Adding @nedbat and @bderusha as reviewers since they were involved with https://github.com/edx/configuration/pull/5691

@nedbat are there any non-edX stakeholders who use head of configuration master that might want to review this PR?

Also, is there a place to write down upgrade notes for Koa.  Since Juniper has the linker assignments merging this PR will mean that teams who have extended configuration will need to do a find and/replace after upgrading to Juniper and before upgrading to Koa.

I am also curious if we should we link this PR to https://github.com/edx/configuration/pull/5691/files, https://openedx.atlassian.net/browse/DEPR-94 or https://discuss.openedx.org/t/deprecation-removal-lowercase-ansible-version-variables-depr-94/2034.

**Comment by adzuci** [comment]
@syedimranhassan it may also be worth reaching out to @bradenmacdonald or @arbrandes to see if this change will impact OpenCraft since they sometimes deploy off of `configuration` master.

**Comment by giovannicimolin** [comment]
Hi @adzuci and @syedimranhassan! Thanks for the ping.

We deploy instances using the `master` branch daily, for periodic check builds and also sandboxes for upstreaming PRs.
I've scheduled a task to review/comment on this PR for early next week.

File: playbooks/legacy_certificates.yml (hunk 1 above)
Comment (syedimranhassan) [reply playbooks/legacy_certificates.yml:11]: currently, we are assigning uppercase value to lower case and then use the lower case everywhere. If we remove the link assignment in separate PR. Then the respective PR will break on sandbox or openedx deployment.

**Comment by syedimranhassan** [comment]
Hi @giovannicimolin 
Any update on this PR review?

**Comment by giovannicimolin** [comment]
@syedimranhassan 

@nizarmah Is the assignee on the task and will provide you with feedback.

**Review by nizarmah — APPROVED** [review]
Yes, @syedimranhassan sorry for the late reply. Passed over during the first week, but needed the team's response to a question I had.

:+1:

> The configuration variables changed to uppercase can be found [here](https://docs.google.com/spreadsheets/d/1uVEWfmEaD4YpcudCMF4EcVSItwR_UEfRUwzPYUxB8MM/edit?usp=sharing), in case it is a use for anyone later on who wishes to deploy off of this repo and needs to do necessary changes after this PR.

**Review by adzuci — APPROVED** [review]
I believe we can merge this after posting a comment to https://discuss.openedx.org/t/deprecation-removal-lowercase-ansible-version-variables-depr-94/2034 linking to this PR and waiting a day or so.  Since it is hard to test there may be a need to revert it, but I think we've done our best to ensure that the right people are aware of the change.

**Pull request merged**